src/css/index.css

```
@@ -14,10 +14,8 @@ body {
 }
 
 #app {
-  margin-top: 56px;
   max-width: 600px;
-  margin-left: auto;
-  margin-right: auto;
+  margin: 56px auto 0 auto;
   padding: 32px 16px 32px 16px;
```

```suggestion
  margin: 56px auto 0;
  padding: 32px 16px 32px;
```
left right 값이 같으면 2번째 값만 적어도 동일한 값이 돼요.

---

3개만 써도 되는지 몰랐네요. 짚어주셔서 감사합니다 :D